docs/1.getting-started/6.server-directory.md

```diff
@@ -0,0 +1,53 @@
+---
+navigation.icon: uil:laptop-connection
+description: Using server directory to create middlewares, plugins and API easily.
+---
+
+# Server Directory
+
+Nuxt scans files under server directory to create middlewares, plugins and API routes to create features to use inside components or pages.
```

Comment:
I think that we could add a first paragraph explaining what is the server directory and how it could be useful

Reply:
How is newly added paragraph? I tried to explain what is it for but might need help about it.

Reply:
Will check it.